pkg/adapterManager/parallelManager_test.go

```diff
@@ -55,12 +56,12 @@ func TestExecute(t *testing.T) {
 			{&configpb.Adapter{Name: c.name}, &configpb.Aspect{Kind: c.name}},
 		}
 
-		o, err := m.Execute(context.Background(), cfg, nil, nil)
-		if c.inErr != nil && err == nil {
-			t.Errorf("m.Execute(...) = %v; want err: %v", err, c.inErr)
+		o := m.Execute(context.Background(), cfg, nil, nil)
```

Comment:
I think we could also make the reasonable argument that due to config validation we shouldn't be hitting golang level errors in adapter execution, but I think we'd have to evaluate that statement later when we've written more adapter code.